ext/reflection/php_reflection.c

```diff
@@ -1110,6 +1110,10 @@ static int read_attributes(zval *ret, HashTable *attributes, zend_class_entry *s
 		return SUCCESS;
 	}
 
+	if (!attributes) {
```

Comment:
This check is already performed in `reflect_attributes()` and the return is unreachable. It should be replaced with `ZEND_ASSERT(attributes != NULL)` and moved to the top instead.